frameworks/hdfs/src/main/dist/core-site.xml

```diff
@@ -32,7 +32,12 @@
         <name>ha.zookeeper.parent-znode</name>
         <value>/{{SERVICE_ZK_ROOT}}/hadoop-ha</value>
     </property>
-
+    {{#TLS_ENABLED}}
+    <property>
+        <name>hadoop.ssl.hostname.verifier</name>
+        <value>ALLOW_ALL</value>
```

Comment:
@jsrodman You are right! I made a typo, it should be `STRICT_IE6`.

The deal with STRICT_IE6 is that it will match either the CN or any of the alternate subject names. Currently, the certificate that's provided to us only contains CN == agent's hostname, and != the task's A-Record.

There's ongoing work to address that, and once it's made available to us, we can move to `STRICT`.